CONTRIBUTING.md

```diff
@@ -128,7 +128,12 @@ The `Fixes:` tags should be put at the end of the detailed description.
 
 You may refer to [How to Write a Git Commit
 Message](https://chris.beams.io/posts/git-commit/) article for
-recommendations for good commit message.
+recommendations for good commit message. Please also add component (for
+instance name of the file you change) prefixes to your subject line like:
+
+```
+crns.py: New attempt to have --unshare option
```

Comment:
I also think that it is a good idea to mention this but it is common for git commits to change multiple files. Using `crns.py` as an example for a prefix could be misleading.

Reply:
I will add one another example here, with non-file prefix.